recordedit/form.controller.js

```diff
@@ -104,6 +104,29 @@
             $log.info(response);
         }
 
+
+        /*
```

Comment:
Just make sure the default for new/unexpected column types is to treat
empty input as NULL.  We don't want this to break every time we add
something like longtext, markdown, etc. in a project.